azure_gateway/_chat.py

```diff
@@ -43,13 +44,19 @@ def to_json(self):
 
 def openai_chat_request(project_id: str, params: OpenAIBody | dict, token: str):
     """
-    Send a request to the specified OpenAI chat API.
+    DEPRECATED: Send a request to the specified OpenAI chat API.
```

Comment:
There is a package for deprecation: https://pypi.org/project/deprecation/

Should we use that here?